assets/js/script.js

```diff
@@ -0,0 +1,195 @@
+  {
+    id: 1,
+    featuredImage: 'assets/images/mobile/portfolio1.png',
+    name: 'Tonic',
+    frame: [
+      'CANOPY', 'Back End Dev', '2015',
+    ],
+    description: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    technologies: [
+      'HTML', 'CSS', 'Javascript',
+    ],
+    year: 2015,
+    liveDemo: '#',
+    sourceDemo: '#',
+  },
+
+  {
+    id: 2,
+    featuredImage: 'assets/images/mobile/portfolio2.svg',
+    name: 'Multi-Post Stories',
+    frame: [
+      'CANOPY', 'Back End Dev', '2015',
+    ],
+    description: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    technologies: [
+      'HTML', 'CSS', 'Javascript',
+    ],
+    year: 2015,
+    liveDemo: '#',
+    sourceDemo: '#',
+  },
+
+  {
+    id: 3,
+    featuredImage: 'assets/images/mobile/portfolio3.svg',
+    name: 'Tonic',
+    frame: [
+      'CANOPY', 'Back End Dev', '2015',
+    ],
+    description: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    technologies: [
+      'HTML', 'CSS', 'Javascript',
+    ],
+    year: 2015,
+    liveDemo: '#',
+    sourceDemo: '#',
+  },
+
+  {
+    id: 4,
+    featuredImage: 'assets/images/mobile/portfolio4.svg',
+    name: 'Multi-Post Stories',
+    frame: [
+      'CANOPY', 'Back End Dev', '2015',
+    ],
+    description: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    technologies: [
+      'HTML', 'CSS', 'Javascript',
+    ],
+    year: 2015,
+    liveDemo: '#',
+    sourceDemo: '#',
+  },
+];
+
+function createProject() {
+  projects.forEach((project) => {
+    const cardDiv = document.createElement('div');
+    cardDiv.className = 'card';
+    const cardImg = document.createElement('img');
+    cardImg.className = 'project1';
+    cardImg.src = project.featuredImage;
+    const cardTextDiv = document.createElement('div');
+    cardTextDiv.className = 'card-text';
+    const cardTextDivH3 = document.createElement('h3');
+    cardTextDivH3.textContent = project.name;
+    const cardFrameDiv = document.createElement('div');
+    cardFrameDiv.className = 'frame-div';
+    for (let i = 0; i < project.frame.length; i += 1) {
+      const cardFrameSpan = document.createElement('span');
+      cardFrameSpan.textContent = project.frame[i];
+      cardFrameDiv.appendChild(cardFrameSpan);
+    }
+    const cardTextDivP = document.createElement('p');
+    cardTextDivP.textContent = project.description;
+    const tagDiv = document.createElement('div');
+    tagDiv.className = 'tag';
+    for (let j = 0; j < project.technologies.length; j += 1) {
+      const tagDivSpan = document.createElement('span');
+      tagDivSpan.textContent = project.technologies[j];
+      tagDiv.appendChild(tagDivSpan);
+    }
+    const createProjectBtn = document.createElement('a');
+    createProjectBtn.className = 'see-proj-btn';
+    createProjectBtn.textContent = 'See Project';
```

Comment:
Buttons are overflowing out of the pop-up on mobile view. Please follow the font guidelines from the design.
![pop-mobile](https://user-images.githubusercontent.com/35504149/137498635-6b44beb0-8497-4aa6-a29a-b48601cacd69.png)